README.md

```diff
@@ -86,17 +222,17 @@ A very good place to ask questions and discuss development work is our
 
 ## Authors
 
-The main author is Raph Levien.
-
-[xi-win]: https://github.com/xi-editor/xi-win
-[winit]: https://github.com/tomaka/winit
-[Synthesizer IO]: https://github.com/raphlinus/synthesizer-io
-[jul-2018-video]: https://www.youtube.com/watch?v=4YTfxresvS8
-[jul-2018-slides]: https://docs.google.com/presentation/d/1aDTRl5R-icAF38Di-qJ4FzAl3pLlutTKVFcr3mUGgYo/edit?usp=sharing
-[2d graphics]: https://raphlinus.github.io/rust/graphics/2018/10/11/2d-graphics.html
-[conrod]: https://github.com/PistonDevelopers/conrod
-[azul]: https://github.com/maps4print/azul
-[ggez]: https://github.com/ggez/ggez
-[CONTRIBUTING.md]: CONTRIBUTING.md
-[Zulip chat instance]: https://xi.zulipchat.com
+The main authors are Raph Levien and Colin Rofls.
```

Comment:
The main authors are Raph Levien and Colin Rofls, with much support from an active community.